apps/provider-console/src/components/dashboard/ResourcesCard.tsx

```diff
@@ -0,0 +1,99 @@
+import { Card, CardContent, CardHeader } from "@akashnetwork/ui/components";
+
+import { formatBytes } from "@src/utils/formatBytes";
+import { StatPieChart } from "./StatPieChart";
+
+const getResourceData = (active: number = 0, pending: number = 0, available: number = 0, isBytes: boolean = false) => {
```

Comment:
number and bool args are generally really hard to understand. so the client code looks like `getResourceData(10, 10, 10, 10, true). Should be an object with properties so the call would look like `getResourceData({ active: 10, isBytes: true.... })

Also the name could be smth like `summariseStatuses`, coz I'm not sure what current one means

Reply:
Done, changed to make it more developer friendly